logical_router_static_route.go

```diff
@@ -158,6 +160,8 @@ func (odbi *ovndb) rowToLogicalRouterStaticRoute(uuid string) *LogicalRouterStat
 			lrsr.OutputPort = []string{outputPort.(libovsdb.UUID).GoUUID}
 		case libovsdb.OvsSet:
 			lrsr.OutputPort = odbi.ConvertGoSetToStringArray(outputPort.(libovsdb.OvsSet))
+		case []string:
```

Comment:
Thanks for the PR! This is a bug. It would be better if the commit message could make it clear.

Question: why for Policy it is "case string:" but for output_port it is "case []string:"? In fact, I think both policy and output_port should just be string type, so we should remove the switch-case.

In addition, it would be better to fix the test case as well to cover this.

Reply:
yes,  policy and output_port should just be string type, but I'm a little confused about why your struct should be set to an array
In fact, i only tested the policy, not the output_port，the next time i will cover this test case

Reply:
The struct definition was a mistake, too. @jxiaobin was working on a fix, which would change the API, but he stopped the progress. Not sure if he will continue working on it soon.

Reply:
I've changed the struct and lrsrdel params, which doesn't seem to require pointers